class/practice10.1/FootballTournament.cpp

```diff
@@ -0,0 +1,257 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <algorithm>
+#include <ranges>
+#include <print>
+
+class Team {
```

Comment:
There is a potential scaling issue here - it is better to store some container of StatisticsPerMatch entities. This will allow to add new functionality. For example history of last few matches, biggest goals count per match and so on. In your implementation you just accumulated all the stat into single variables and after that there will be no way to extract that info from team object